### DISCUSSION TITLE
[feature request] possible to list uncommented discussions

### DISCUSSION THREAD

**Discussion by dinsen** [community/community#3377] [Discussions]
Hi

It's possible to list answered and unanswered discussions, but I can't find a way to show only discussions that don't have any comments/replies? 

If it's not possible, you have a feature request 😃

**Comment by PranamBhat** [comment]
Hello!!

Yes. It is available.

If you click on whichever filter you are looking for, then it will list out discussions as per the filter selections.

Now if you notice on the right side - It will show the counts (number of comments in the particular discussion).

![image](https://user-images.githubusercontent.com/32120033/108152623-6ba6d880-70ff-11eb-959c-92c1a7287938.png)

**Reply by dinsen** [reply]
Hi @PranamBhat 

I know the Answered and Unanswered buttons. I just want an easier way to list all discussions that have 0 comments so i'm sure that every discussion will get at least 1 answer.

**Reply by PranamBhat** [reply]
Got your point.

But as we can see in **GitHub** Discussion page, there's already lot of filters at the top. If GitHub keep adding again few more filter's, it will be a total mess!
